webapp/src/main/webapp/WEB-INF/conf/news/portal-configuration.xml

```diff
@@ -35,5 +35,73 @@
         </object-param>
       </init-params>
     </component-plugin>
+    <component-plugin>
+      <name>new.portal.config.user.listener</name>
+      <set-method>initListener</set-method>
+      <type>org.exoplatform.portal.config.NewPortalConfigListener</type>
+      <description>this listener delete some predefined portal and templates configuration</description>
+      <init-params>
+        <object-param>
+          <name>site.templates.location</name>
+          <description>description</description>
+          <object type="org.exoplatform.portal.config.SiteConfigTemplates">
+            <field name="location">
+              <string>war:/conf/news/portal</string>
+            </field>
+            <field name="portalTemplates">
+              <collection type="java.util.HashSet">
+                <value>
+                  <string>empty</string>
+                </value>
+              </collection>
+            </field>
+          </object>
+        </object-param>
+        <object-param>
+          <name>portal.configuration</name>
+          <description>description</description>
+          <object type="org.exoplatform.portal.config.NewPortalConfig">
+            <field name="predefinedOwner">
+              <collection type="java.util.HashSet">
+                <value>
+                  <string>global</string>
+                </value>
+              </collection>
+            </field>
+            <field name="ownerType">
+              <string>portal</string>
+            </field>
+            <field name="location">
+              <string>war:/conf/news/portal</string>
+            </field>
+            <field name="override">
+              <boolean>${exo.ecms.portalConfig.metadata.override:true}</boolean>
+            </field>
+            <field name="importMode">
+              <string>${exo.ecms.portalConfig.metadata.importmode:insert}</string>
+            </field>
+          </object>
+        </object-param>
+        <object-param>
+          <name>group.configuration</name>
+          <description>description</description>
+          <object type="org.exoplatform.portal.config.NewPortalConfig">
+            <field name="predefinedOwner">
+              <collection type="java.util.HashSet">
+                <value>
+                  <string>/platform/web-contributors</string>
+                </value>
+              </collection>
+            </field>
+            <field name="ownerType">
+              <string>group</string>
+            </field>
+            <field name="location">
+              <string>war:/conf/news/portal</string>
+            </field>
+          </object>
+        </object-param>
```

Comment:
you need only this object param to be added in the above NewPortalConfigListener